### ISSUE TITLE
Custom truststore for LDAPS

### ISSUE THREAD

**Issue by Roemer** [SonarSource/docker-sonarqube#367]
Hello
I am trying to get LDAPS running. For this, I created a custom truststore file and placed it in the fle `/opt/sonarqube/conf/MyTrustStore`. The trust store works as I use the same for Jenkins.
Now I fail to tell sonar to use this truststore.
I tried by setting
```
sonar.web.javaAdditionalOpts=-Djavax.net.ssl.trustStore=/opt/sonarqube/conf/MyTrustStore -Djavax.net.ssl.trustStorePassword=<password>
```
But it fails to initialize. I think the problem is that the truststore now is only used for the web server but it should be used for the core-server (if that exists).
Other java opts are for the Computing Engine or the search, which also does not fit.

Is this possible without creating a custom Dockerfile which just appeds (or replaces) the existing truststore?

Thanks for your help.

**Comment by Roemer** [comment]
Somehow the `sonar.web.javaAdditionalOpts` seems not be be added to the startup of the web.
If I add all those parameters to `sonar.web.javaOpts` it does work as expected.
Do I missunderstand something about `sonar.web.javaAdditionalOpts` or is this a bug?

**Comment by pierre-guillot-sonarsource** [comment]
Hello, which tag image are you using ?

**Comment by Roemer** [comment]
7.9.2-community

**Comment by pierre-guillot-sonarsource** [comment]
https://jira.sonarsource.com/browse/SONAR-12501 bug includes `javaAdditionalOpts`. This was fixed with 8.0. If you can't upgrade, consider putting this property directly in the sonar.properties file

**Comment by Roemer** [comment]
Ah thanks for the info. Will try to upgrade or use the javaOpts which works.
